redis/src/cmd.rs

```diff
@@ -473,6 +473,35 @@ impl Cmd {
         (self.args.capacity(), self.data.capacity())
     }
 
+    /// Clears the command, resetting it completely.
+    ///
+    /// This is equivalent to [`Cmd::new`], except the buffer capacity is kept.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// # use redis::{Client, Cmd};
+    /// # let client = Client::open("redis://127.0.0.1/").unwrap();
+    /// # let mut con = client.get_connection().expect("Failed to connect to Redis");
+    /// let mut cmd = Cmd::new();
+    /// cmd.arg("SET").arg("foo").arg("42");
+    /// cmd.query::<()>(&mut con).expect("Query failed");
+    /// cmd.clear();
+    /// // This reuses the allocations of the previous command
+    /// cmd.arg("SET").arg("bar").arg("42");
+    /// cmd.query::<()>(&mut con).expect("Query failed");
+    /// ```
+    pub fn clear(&mut self) {
+        self.data.clear();
+        self.args.clear();
+        self.cursor = None;
+        self.no_response = false;
+        #[cfg(feature = "cache-aio")]
+        {
+            self.cache = None;
+        }
```

Comment:
It's cleaner without brackets
```suggestion
        #[cfg(feature = "cache-aio")]
        self.cache = None;
```

Reply:
Doesn't work:
```
    Checking redis v0.30.0 (/home/kriskras99/Source/redis-rs/redis)
error[E0658]: attributes on expressions are experimental
   --> redis/src/cmd.rs:499:9
    |
499 |         #[cfg(feature = "cache-aio")]
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #15701 <https://github.com/rust-lang/rust/issues/15701> for more information
    = help: add `#![feature(stmt_expr_attributes)]` to the crate attributes to enable
    = note: this compiler was built on 2025-05-10; consider upgrading it if it is out of date

For more information about this error, try `rustc --explain E0658`.
error: could not compile `redis` (lib) due to 1 previous error
```